sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/NotificationHandler.groovy

```diff
@@ -22,8 +22,8 @@ class NotificationHandler {
         subscribeToService()
     }
 
-    private static void showMessage(String message, Type type) {
-        Notification notification = new Notification(message, type)
+    private void showMessage(String message, Type type) {
```

Comment:
I think this still can be `static`. There is no reason why this functionality is coupled to the current status of the NotificationHandler.